### PR TITLE
[master < T0860-MG] Add emsdk 3.9 instead of latest in wasm install deps

### PR DESCRIPTION
**Pull request by kostasrim**

### CODE REVIEW

**Review by jbajic — APPROVED** [review]

File: wasm/install_deps.sh [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 git clone https://github.com/emscripten-core/emsdk.git ${DIR}/emsdk
-${DIR}/emsdk/emsdk install latest
-${DIR}/emsdk/emsdk activate latest
+${DIR}/emsdk/emsdk install sdk-releases-upstream-edabe25af34554d19c046078f853999b074259ca-64bit
```
Comment (antaljanosbenjamin) [inline wasm/install_deps.sh:5]: https://emscripten.org/docs/getting_started/downloads.html#emsdk-install-targets

What about this?
```suggestion
${DIR}/emsdk/emsdk install 3.1.9
```

Or do I missmatch something?

Comment (kostasrim) [reply wasm/install_deps.sh:5]: Already tried but it doesn't work with this tag :/

Comment (kostasrim) [reply wasm/install_deps.sh:5]: After discussing this on slack, it turns out I was using an older version locally which `3.1.9` was not available. I will patch this

**Review by antaljanosbenjamin — APPROVED** [review]
LGTM

**Pull request merged**